cockatrice/src/game/zones/view_zone_widget.cpp

```diff
@@ -46,27 +46,43 @@ ZoneViewWidget::ZoneViewWidget(Player *_player,
         QGraphicsLinearLayout *hPilebox = new QGraphicsLinearLayout(Qt::Horizontal);
         QGraphicsLinearLayout *hFilterbox = new QGraphicsLinearLayout(Qt::Horizontal);
 
-        QGraphicsProxyWidget *sortByNameProxy = new QGraphicsProxyWidget;
-        sortByNameProxy->setWidget(&sortByNameCheckBox);
-        hFilterbox->addItem(sortByNameProxy);
-
-        QGraphicsProxyWidget *sortByTypeProxy = new QGraphicsProxyWidget;
-        sortByTypeProxy->setWidget(&sortByTypeCheckBox);
-        hFilterbox->addItem(sortByTypeProxy);
+        // groupBy options
+        groupBySelector.addItem(tr("Group by ---"), CardList::NoSort);
```

Comment:
if tr() is outside of retranslateui() then when a user changes the language in the settings while this string is out it will not be updated

Reply:
Yeah. Unfortunately, I'm not sure how to get `retranslateUi` to work with strings that are already inside a `QComboBox`.  I checked around and I don't think we ever handle the combination of `retranslateUi` and `QComboBox`s